core-java-modules/core-java-collections-5/src/test/java/com/baeldung/immutables/ImmutableCollectionsUnitTest.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.immutables;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.Test;
```

Comment:
Sorry for having not realized this before, could you update these imports so we use JUnit 5 here.

Reply:
I have updated it accordingly